scripts/test-data/gen-fixtures.js

```diff
@@ -109,24 +113,54 @@ const HEADER = `/**
 `;
 
 try {
+	// Article fixtures
 	const requests = articles.map((article) => {
 		return fetch(`${article.url}.json?dcr`)
 			.then((res) => res.json())
 			.then((json) => {
 				// Add test config
 				json.config = config;
+				// TODO: Remove this when we add in support for CAPI format to DCR
+				delete json.format;
```

Comment:
This needs to be removed when @buck06191's change is brought in

Reply:
https://github.com/guardian/dotcom-rendering/pull/2634 is the one to watch